source/platforms/coiled.md

```diff
@@ -1,65 +1,142 @@
+
+```bash
+coiled login
 ```
 
-For more information see the [Coiled Getting Started documentation](https://docs.coiled.io/user_guide/getting_started.html).
+For more information see the [Coiled Getting Started documentation](https://docs.coiled.io/user_guide/setup/index).
 
-### Software Environment
+## Notebook Quickstart
 
-Next you'll need to register a RAPIDS software environment with Coiled.
+The simplest way to get up and running with RAPIDS on Coiled is to launch a Jupyter notebook server using the RAPIDS notebook container.
 
-You can either build this from the official RAPIDS Container images.
+```bash
+coiled notebook start --gpu --container {{ rapids_notebooks_container }}
+```
 
-```python
-import coiled
+![Screenshot of Jupyterlab running on Coiled executing some cudf GPU code](../_static/images/platforms/coiled/coiled-jupyter.png)
 
-coiled.create_software_environment(
-    name="rapids-{{ rapids_version.replace(".", "-") }}",
-    container="{{ rapids_container }}",
-)
+## Software Environments
+
+By default when running remote operations Coiled will [attempt to create a copy of your local software environment](https://docs.coiled.io/user_guide/software/sync.html) which can be loaded onto the remote VMs. While this is an excellent feature it's likely that you do not have all of the GPU software libraries you wish to use installed locally. In this case we need to tell Coiled which software environment to use.
+
+### Container images
+
+All Coiled commands can be passed a container image to use. This container will be pulled onto the remote VM at launch time.
+
+```bash
+coiled notebook start --gpu --container {{ rapids_notebooks_container }}
 ```
 
-Or you can create it with a list of conda packages in case you want to customize the environment further.
+This is often the most convenient way to try out existing software environments, but is often not the most performant due to the way container images are unpacked.
 
-```python
-import coiled
-
-coiled.create_software_environment(
-    name="rapids-{{ rapids_version.replace(".", "-") }}",
-    gpu_enabled=True,  # sets CUDA version for Conda to ensure GPU version of packages get installed
-    conda={
-        "channels": {{ rapids_conda_channels_list }},
-        "dependencies": {{ rapids_conda_packages_list + ["jupyterlab"] }},
-    },
-)
+### Coiled Software Environments
+
+You can also created Coiled software environments ahead of time. These environments are built and cached on the cloud and can be pulled onto new VMs very quickly.
+
+You can create a RAPIDS software environment using a conda `environment.yaml` file or a pip `requirements.txt` file.
+
+#### Conda example
+
+Create an environment file containing the RAPIDS packages
+
+```yaml
+# rapids-environment.yaml
+name: rapidsai-notebooks
+channels:
+  - rapidsai
+  - conda-forge
+  - nvidia
+dependencies:
+  # RAPIDS packages
+  - rapids={{ rapids_version }}
+  - python=3.12
+  - cuda-version>=12.0,<=12.5
```

Comment:
```suggestion
```

`rapids` already has a `run:` dependency on compatible versions of `cuda-version`: https://github.com/rapidsai/integration/blob/172ef624ea50670969e1fd79930a46eabdd9c3c9/conda/recipes/rapids/meta.yaml#L31

And which packages you get should happen nicely and automatically based on the detected CUDA version on the system.

I think we could probably safely remove this. That'd also mean one less thing that's likely to become out of date over time, as RAPIDS support matrix of CUDA versions shifts.

Reply:
This environment file gets built on a build server which AFAIK doesn't have GPUs. They just set the cuda package, but it's not clear what they set it to. So if we omit this constraint I'm not sure what will happen. What do you think?

```console
$ coiled env create --help
...
  --gpu-enabled                   Set CUDA virtual package for Conda
...
```